app/assets/javascripts/index/element.js

```diff
@@ -1,10 +1,16 @@
 (function () {
+  let abortController = null;
+  const langs = [...new Set([...OSM.preferred_languages.map(l => l.toLowerCase()), "mul", "en"])];
+  const wikis = [...new Set(langs.map(l => l.split("-")[0] + "wiki"))];
```

Comment:
Wikimedia wiki codes don’t quite conform to ISO&nbsp;639. For example, the response will include `be_x_oldwiki` for what this repository calls `be-Tarask`. But this will affect only a handful of locales that we have localizations for, so it probably isn’t critical.